<launch>
  <arg name="rslidar_config_file" default="$(find rslidar_sdk)/config/config.yaml"/>
  <param name="rslidar_config_file" value="$(arg rslidar_config_file)"/>
  <param name="ros_send_point_cloud_topic" value="points"/>
  <param name="lidar_type" value="RSM1"/>
  <node pkg="rslidar_sdk" name="rslidar_sdk_node" type="rslidar_sdk_node" output="screen">
  </node>
  <!-- rviz -->
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find rslidar_sdk)/rviz/rviz.rviz" />
</launch>
